<h2><em>Casting a wide investigative net  to define microbial behaviors that lead to periodontitis</em></h2>

<div class="callout" style="font-style: normal; font-variant: normal; font-weight: normal; font-stretch: normal; font-size: 1.6em; line-height: 1.4em; font-family: georgia, &#39;times new roman&#39;, serif; color: rgb(45, 123, 19); padding: 10px 15px 0px 20px; margin: 0px 0px 1.5em 20px; float: right; width: 208px; background-color: white;"><p><em>NIDCR grantee Dr. Jorge Frias-Lopez discusses his results in harnessing genomic technology to uncover the molecular profiles of periodontitis </em>
<br/><a href="#QA" class="ms-rteFontSize-2 ms-rteThemeFontFace-2"><span style="text-decoration: underline;">Jump to Q&amp;A »</span></a></p></div>

<p>In the early 1980s, Americans turned on the​ir television  sets at night and settled in for sit-coms with predictable plots and  regrettable fashions.  Like now, every 15  minutes the networks cut to commercials to laud products and build a better  you.  Slotted among the better yous was  Madison Avenue’s first crack at introducing American consumers to a previously  unpublicized threat to their oral health:   Gingivitis.</p>
<p><em>Authoritative male  voice</em>:   “When you ignore plaque,  you’re gambling with gingivitis, an early form of gum disease.” The winning  bet:  A dental product that specifically  “kills germs that cause plaque and gingivitis.” </p>
<p>While the commercials made sense – practicing good oral  hygiene is always a good idea – they raised a few questions. Among them was  which germs <em>really</em> needed to be  killed to prevent gingivitis and its possible advance to tooth-threatening periodontitis?  At the time, researchers were divided into  two schools of thought.  The <em>specific-plaque</em> camp placed blame  squarely on a handful of bad germs (pathogens) that live in the sticky plaque  that forms along and under the gumline, known as the supragingival and  subgingival spaces.  Their data suggested  that the pathogens expanded their numbers and geographical footprint over time  to infect nearby cells and trigger gingivitis.   The <em>non-specific-plaque</em> camp  took the opposite view. They hypothesized that some as-yet undetermined dark  factor specifically within the subgingival space caused the resident microbes,  collectively known as the oral microbiota, to overgrow their normal boundaries  and infect tissues of the mouth. </p>
<p>That’s where the science and Madison Avenue stood until in  the early 1990s when the British microbiologist Philip Marsh split the difference  between the two theories and proposed a new hypothesis.  He wrote, “Some of the arguments surrounding  these hypotheses may be, in part, about semantics (e.g. the definition of  ‘specific’ or ‘non-specific’), since plaque-mediated diseases, while not  necessarily having a totally specific etiology, do show evidence of  specificity.” </p>
<h4>The  Dynamic Oral Microbial Community</h4>
<div style="float: right; width: 220px; padding-left: 10px;">
  <img width="200" alt=" Microbial Communities in Dental Plaque" src="/imagegallery/science/PublishingImages/science045.jpg" style="padding-top: 10px; padding-bottom: 10px;"/>
  <p style="width: 200px; color: rgb(34, 34, 34); font-size: 11px;"><strong>Fluorescence microscopy shows multiple species of bacteria in dental plaque</strong></p></div>

<p>Marsh then unified the best of both theories to propose his<em> ecological plaque hypothesis</em>.  He argued that dominant microbes might cause  periodontal disease, but they do so only if a specific environmental  catastrophe within the plaque unleashes their ill behavior.  “It is a basic tenet of microbial ecology  that a major change to an ecosystem produces a corresponding disturbance to the  resident community,” he explained.   “Examples of this have been reported in ecosystems as diverse as soil,  skin, and water.” <br/>

</p>
<p>As Marsh’s hypothesis gained adherents in the 1990s,  periodontal research adapted a number of existing ecological terms.  Most notably, dental plaque became a <em>biofilm</em>, a dynamic, polymicrobial  community that, like lichens growing along a craggy coast, colonizes various  and sundry pits, fissures, and other oral surfaces.  From this ecological perspective, the way  forward in periodontal research involved gaining a more detailed understanding  of these microbial ecosystems, the environmental disturbances that destabilize  them, and the opportunistic pathogens that exploit the imbalance to cause  disease.</p>
<h4>DNA  Technology and the Oral Biofilm</h4>
<p>
By the late 1990s and into the early 2000s, sequencing DNA  had become technically feasible and more affordable. The NIDCR and other  research funding agencies hurried to support multi-lab efforts that assembled  the first full-sequence DNA maps of several key oral pathogens.  These highly anticipated new resources meant  that researchers for the first time could pore over the genetic blueprints  (genomes) of these bugs and perform comparative DNA analyses across species  (genomics) to generate more strategic, evolution-based hypotheses about what  made a bad bug bad.  Because genomics  brought its own unique investigative perspective and language to biology, the  periodontal lexicon underwent another predictable shift.  Most notably, the freshly minted genomic term <em>microbiome</em> gave microbiota a sudden run  for its investigative money.  Initially,  the new M word referred to the collection of microbial genomes in a given  ecosystem (tooth, rock, plastic), shifting the investigative focus from the  bugs themselves to the A, C, G, Ts in their genetic codes. The focus eventually  dialed back to the current more-general connotation: all of the microbes in a  biofilm, with an emphasis on their genomic information. </p>
<p>While the microbiome meme entered through one window, a new  generation of genomic technologies arrived through another. These powerful  laboratory tools allowed scientists to pursue an emerging scientific discipline  called <em>metagenomics</em>.  It builds on the premise that a biofilm is  not a hodgepodge of microbial tenants bottled up in a sticky matrix but a  single, collective, living organism.  To  study this community, scientists could extract the genomes from the various  microbes living in a given biofilm and then sequence them to compile a working  inventory of all the microbial genes present in the organism.  </p>
<div class="callout" style="font-style: normal; font-variant: normal; font-weight: normal; font-stretch: normal; font-size: 1.6em; line-height: 1.4em; font-family: georgia, &#39;times new roman&#39;, serif; color: rgb(45, 123, 19); padding: 10px 15px 0px 20px; margin: 0px 0px 0px 20px; float: right; width: 208px; background-color: white;"><p><em>“The oral biofilm is not a hodgepodge of  microbial tenants bottled up in a sticky matrix but a single, collective,  living organism.”</em></p></div>
<p>For a typical oral biofilm, which houses up to hundreds of  distinct microbial species, the inventory yields a catalogue of genes that  easily can stretch into the hundreds of thousands. This opened two  investigative doors.  One, this body of  biological information could be entered into a searchable online database as a <em>bioinformatic</em> tool to generate better  working hypotheses and enable more targeted future studies.  Two, these inventories were generated from  biofilms living in their natural environments, not cultured under artificial  conditions in a laboratory.  It was the  difference between learning French on the streets of Paris and in a  college-language lab.</p>
<p>This new ability to inventory biofilms brought another shift  in terminology.  Instead of putting a  bolder, genome-friendly coat of paint on the 1970s term biofilm, scientists  opted for the modern <em>metagenome</em> as a  synonym for microbiome to emphasize that the data emanate  from DNA taken directly from a natural environment in the course of performing  metagenomics.</p>
<p>But metagenomes came to the investigative fore with inherent  limitations.  Because polymicrobial  communities are complex, DNA collection can be inefficient, some DNA fragments  are easier to sequence than others and can possibly introduce bias, and the  required sequencing capacity to churn out the data can be enormous. In  addition, although the genetic code contained in metagenomes can provide broad  hints about the possible activities performed in a microbiome in its natural  environment, it offers no direct evidence that these activities actually take  place in time (when) and space (where). </p>
<h4>Uncovering  Gene Activity in the Oral Biofilm</h4>
<p>
  To tap more directly into the life of the microbiome,  researchers learned to harvest messenger RNA (mRNA) from microbiomes, convert  it back to DNA, and sequence it. What’s mRNA?   When the information encoded in a gene is expressed, the DNA is copied  chemically into an mRNA molecule, an information-bearing intermediary that will  be translated elsewhere in the cell as the directions to make a specific  protein. By sequencing the information, scientists are in a position to  intercept and unencrypt what amounts to a biological order form to make a  protein.  When viewed across the myriad  bugs layered into a microbiome, mRNA (<em>metatranscriptome</em>)  serves as a measurable record of which of the hundreds of thousands of genes  present in this multi-layered microbial community are actually expressed at any  one time.  Or, more tellingly,  metatransciptomes can serve as a before-and-after record of how gene expression  shifts in response to environmental changes, such as an increased acidity or  mechanical stress. </p>

  <p>
  The metatranscriptome brought an important new investigative  wrinkle to the study of periodontal disease.   Scientists could develop methods to harvest mRNAs and as comprehensively  as technically possible from the subgingival spaces of people in good oral  health and those with periodontitis.   After sequencing the most abundant transcripts, they would have genomic  snapshots that provide for the first time composite pictures to compare  functionally what stays right for some and goes wrong for others. Or, they  could take their analyses to the next conceptual level.  Instead of trying to identify who is there by  name, scientists can focus on the collective tasks, or functions, that oral  microbiomes tend to perform in periodontal sickness and health.  It’s much like walking down a busy city  street. While nodding and greeting everyone by name would be great, it’s more  important to your safety to be aware of your surroundings and monitor the  behaviors, good and bad, of those congregating on the street.  </p>
<p>This brings us back to theory. Because the study of the oral  microbiome has improved by technological leaps and bounds in recent years,  scientists have made headway parsing the specificity of the earlier plaque  hypotheses and modelling their discoveries within Marsh’s ecological plaque  theory.</p>
<h4>Big  Data and Periodontitis</h4>
<p>
The modeling suggests microbes that were labelled decades  ago as periodontitis-causing pathogens might be more disruptive in the  subgingival space rather than overtly pathogenic.  For example, the oral bacterium <em>P. gingivalis</em>, long suspected of causing  periodontitis, appears in reality to have mastered how to paralyze a key  component of our natural immunity.  Less  efficient community policing in the subgingival space creates an opportunity  for the usually law-abiding (<em>commensal</em>)  residents to take advantage of the anything-goes dynamic to grow, turn  opportunistically pathogenic, and cause periodontitis. This pathogenic tipping  point takes us to one last term, which reflects the coalescence of microbiome  research throughout the body.  The term,  coined by German gastroenterologists in the late 1950s, is <em>dysbiosis</em>. </p>
<p>“[Dysbiosis] can be likened, perhaps, to that of a crew  team,” wrote Drs. George Hajishengallis and Richard Lamont recently of this  dysbiotic microbial synergism that seems to underlie periodontitis.  “All the oars need to be manned but the  identities of individual crew members, provided they are capable of rowing, are  not important for forward progression. Some functions, such as cox, are  important for coordinating characteristics such as direction and speed.”</p>
<p>Now compelling metagenomic and metatranscriptomic data have  begun to enter the periodontitis research literature.  Among the most fascinating is a <a href="http://www.ncbi.nlm.nih.gov/pubmed/24599074" target="_blank" style="text-decoration: underline;">paper</a> published in the  August 2014 issue of the <em>ISME Journal</em> by a team of researchers at the <a href="http://forsyth.org/" target="_blank" style="text-decoration: underline;">Forsyth Institute</a> in Cambridge, MA.  The NIDCR-funded team compared the metagenomes  and metatranscriptomes of several people with chronic periodontitis to those of  healthy volunteers.  The results, though  preliminary, provide a fascinating glimpse into progression. In a follow-up <a href="http://www.ncbi.nlm.nih.gov/pubmed/24866802" target="_blank" style="text-decoration: underline;">article</a>, published in  the August issue of the journal <em>Infection  and Immunity</em>, the scientists found a needle in their many haystacks of data  to generate a new lead in understanding how <em>P.  gingivalis</em> can disrupt a healthy biofilm.</p>

<a name="QA"></a><h2><em>A Q&amp;A with Dr.  Jorge Frias-Lopez</em></h2>

<div style="margin: 0px 30px 20px 0px; width: 244px; float: left; clear: both;">
<img width="244" height="235" alt="Dr. Jorge Frias-Lopez" src="/ScienceSpotlight/InterviewsbyTopic/PeriodontalDisease/PublishingImages/dr-jorge-frias-lopez.jpg"/>
<p style="padding: 4px 0px 0px; margin: 0px; color: rgb(34, 34, 34); font-size: 11px;"><strong>
Dr. Jorge Frias-Lopez</strong> <br/><em>The Forsyth Institute, Cambridge, Massachusetts</em></p>
</div>


<p>The Science Spotlight recently spoke with <a href="http://forsyth.org/person/scientist/jorge-frias-lopez" target="_blank" style="text-decoration: underline;">Jorge Frias-Lopez,  Ph.D</a><span style="text-decoration: underline;">.</span>, a microbial ecologist at Forsyth and a senior author on both  papers.  He offered some perspective on  his findings and the larger picture of genomics as an engine of discovery for  periodontal disease. </p>
<div></div>
<p style="color: rgb(27, 105, 1); font-weight: bold; font-size: 1.2em; line-height: 1.3em;">Let’s start  with the end in mind.  If I were a  periodontist, why should I keep my eye on metagenomes and  metatranscriptomes? </p>
<p>Virtually every periodontist has wondered why some sites in  the mouth remain stable and others progress to advanced disease?  Same patient, same oral cavity, different  biofilms.  Ideally, periodontists could  probe a suspicious site during an examination, characterize its microbial  content, and determine whether another episode of periodontal disease will  occur.  Right now, that type of  diagnostic specificity isn’t possible.   Even if your dentist had a point-of-care technology handy to attempt it,  the test results would be meaningless.   The science simply isn’t there to program a point-of-care device on what  to look for, when to look for it, and why.  </p>
<p>Metagenomics and metatranscriptomics are the first steps in  determining the what, the when, and the why that will program that future  diagnostic device.  The work builds on  biology’s phenomenal progress in genomics over the past few decades and now  allows research groups like the one here at Forsyth to cast a wide  investigative net and haul in vast amounts of the messenger RNA produced by  oral biofilms at all stages of the disease process.  By hauling in these raw data sets, it is  possible to profile the underlying biology of progression and ascertain the  functions that seem to drive the process.  </p>
<p style="color: rgb(27, 105, 1); font-weight: bold; font-size: 1.2em; line-height: 1.3em;">By functions,  you mean collective behaviors, not pinning the blame on individual  micro-organisms?</p>
<p>Yes.  Rather than  saying, these two organisms are usually prevalent and must be up to no good, I  want to know which set of communal activities defines the potential for  progression at a specific site.  I think function  ultimately is going to be more relevant to explain progression than who lives  there.  Sure, knowing who lives there and  the genes encoded in their genomes offers a glimpse into their potential to do  certain things.  But potential doesn’t  tell you anything about what’s really happening in the biofilm.</p>
<p style="color: rgb(27, 105, 1); font-weight: bold; font-size: 1.2em; line-height: 1.3em;">Let’s go to  your paper and see what’s happening.   Your group collected supra- and subgingival plaque samples from six  healthy volunteers and seven people with severe periodontitis.  Each sample was deep-sequenced.  By deep sequenced, I’m referring to a  next-generation sequencing strategy that gives a greater depth of coverage to  identify the sequences contained in the RNA molecules suspended in the biofilm  and their frequency of expression.  Just  to give readers an idea of the scale – no, avalanche - of data generated, what  were the yields?</p>
<p>Our yields from participants ranged from 400,000 to 20  million sequences for the metagenome alone.   For the metatranscriptome, the yields ranged from 900,000 to 39 million  sequences per participant. Now, of the 1,278,494 genes identified via the deep  sequencing and which already were entered into a genomic reference database,  702,106 had at least one hit in more than one sample. </p>
<p style="color: rgb(27, 105, 1); font-weight: bold; font-size: 1.2em; line-height: 1.3em;">That’s roughly  equivalent to the population of Detroit?</p>
<p>That’s right. You treat your sample as though it were a  single organism, in much the same way that Detroit is the sum of its many  neighborhoods.  From this perspective, we  wanted to know what all of those 702,106 genes were doing.  The deep sequencing gave us the genetic code  and frequency of expression.  That’s one,  vital side of the coin.  The flip side of  the coin is bioinformatics.  I created a  huge database with all of the genomes that are available to the oral  community.  Once you have the  transcription profiles from the deep sequencing, you try to map those  transcripts to your database.  The  database allows you to figure out which gene is expressed and its phylogenetic  origin.  In other words, our results are  only as good as our database.</p>
<p style="color: rgb(27, 105, 1); font-weight: bold; font-size: 1.2em; line-height: 1.3em;">And at this  point, it’s pretty good?</p>
<p>I think so.  As of  now, the database has 312 species and more than 500 genomes.  There may be some organisms that are  important, and we don’t have their genomes yet.   In the paper’s supplementary section, we found that 95 percent of the  sequences from an “in silico” community match the right gene in our  database.  What’s even better, only 0.1  percent went to the wrong place. So, that’s the error rate that you could have  in your analysis.  </p>
<p style="color: rgb(27, 105, 1); font-weight: bold; font-size: 1.2em; line-height: 1.3em;">But I noticed  in the paper that your group reports on behavior and the micro-organisms  themselves. If the group behavior is what will matter most in defining  progression of periodontitis, why straddle both sides of the investigative  fence?</p>
<p>Good point.  This is  our first publication to present our community-wide analyses.  We thought it was important to weave in as a  common frame of reference the likely sources of a pathogenic behavior within  the group dynamic.</p>
<p style="color: rgb(27, 105, 1); font-weight: bold; font-size: 1.2em; line-height: 1.3em;">What’s  interesting is this common frame of reference allows your group to present in  the paper something old and something new.   Let’s start with the old, and I’m referring here to the Red  Complex.  What is the Red Complex?</p>
<p>In the late 1980s, the late Sig Socransky and late Anne  Haffajee here at the Forsyth Institute performed seminal work that allowed them  to catalogue into a color-coded system the consortia of dominant bugs  associated with periodontal health, gingivitis, and periodontitis.  The Red Complex describes the latter and  consists of three oral pathogens - <em>Porphyromonas  gingivalis</em>, <em>Treponema denticola</em>,  and <em>Tannerella forsythia</em>.  The key word is associated.  At the time, the field lacked the needed  tools to drill down and look at function and progression. </p>
<p style="color: rgb(27, 105, 1); font-weight: bold; font-size: 1.2em; line-height: 1.3em;">One way to  measure pathogenicity – and which Socransky and Haffajee couldn’t measure at  the time - is the production of virulence factors.  In Figure 5 of your paper, 64 species are  ranked by the number of putative virulence factors that each contributes to the  metatranscriptome.  How does the Red  Complex fare? </p>
<p>Quite well actually.  <em>P. gingivalis</em> and <em>T. denticola</em> ranked in the top five, and <em>T. forsythia</em> comes in fairly high on the list.  This was very reassuring for us. You would  expect to find the pathogens that everybody talks about to be near the top in  producing virulence factors.  If they  didn’t make the cut, I would have been suspicious that something was wrong with  our analysis.  But the Red Complex was  there, and so were a number of new organisms.   So, all in all, we saw some really interesting things.  I think we have a solid base now to proceed.  We can be fairly sure that our results  reflect what’s really going on during severe periodontitis. </p>
<p style="color: rgb(27, 105, 1); font-weight: bold; font-size: 1.2em; line-height: 1.3em;">What was most  interesting about the new organisms?</p>
  <p>
  There were a lot of organisms that are not considered at  this point to be periodontal pathogens that were producing large numbers of  putative virulence factors.  I say,  “putative” because this is based on homology.  </p>
<p style="color: rgb(27, 105, 1); font-weight: bold; font-size: 1.2em; line-height: 1.3em;">Meaning the  gene sequences, to various degrees, resemble known virulence factors?</p>
<p>Right.  It may turn  out that not all of them are virulence factors.   But the huge number of them being produced suggests to me that the whole  community is becoming more virulent, not just the Red Complex organisms.  Somehow the new environment induces the  production of these virulence factors in the whole community. It’s just a  competition to make them.  To me, that  was the most surprising and interesting.   It indicated that the community becomes more dysbiotic and more virulent  under those environmental conditions.  </p>
<p style="color: rgb(27, 105, 1); font-weight: bold; font-size: 1.2em; line-height: 1.3em;">Bacteria,  though, aren’t the only occupants of oral biofilms.  What about archaea, the single-celled  organisms that are one of the proposed three domains on the tree of life?    </p>
<p>We saw some archaea, but their numbers and activities were  at a really low level.  I’ve referenced  other papers that mention this.  Archaea  are there, but they are never in large numbers.   I should note that we didn’t look for fungi, although it’s certainly  possible to do so.  </p>
<p style="color: rgb(27, 105, 1); font-weight: bold; font-size: 1.2em; line-height: 1.3em;">What about  viruses?</p>
<p>No, and that surprised me.   But my colleague and co-author on the paper Ricardo Teles mentioned that  to see expression, you must catch viruses at the exact moment that they are  replicating.  Viruses are not like  bacteria.  Either they are there doing  nothing.  Or, they are actually  replicating in high numbers.  If you  don’t take your sample right at the moment that they are active, you won’t see  them. </p>
<p style="color: rgb(27, 105, 1); font-weight: bold; font-size: 1.2em; line-height: 1.3em;">Your paper  lists several other community functions in addition to virulence factors.  Let’s run through some of them.  You saw a lot of genes involved in iron  acquisition.  What is it about iron?</p>
<p>Iron is extremely limited in nature.  A large number of pathogens need iron to  grow.  For many, it is a driving force in  their life cycle.  So, it wasn’t surprising  that we saw so many genes expressed that are involved in iron acquisition.  Actually, the data were reassuring.  What I mean is although our data represent a  snapshot in time, they show that what we expected was happening.  That’s a real advantage of our metagenomic  approach.  You don’t see this type of  activity in the lab under artificial conditions.  The one surprise here was the proportion of  genes, for example, in the Red Complex, that were involved in this  function.  These bacteria spend a LOT of  energy at the point of disease to take in iron.  </p>
<p style="color: rgb(27, 105, 1); font-weight: bold; font-size: 1.2em; line-height: 1.3em;">During  periodontitis, you found that a lot of antibiotic-resistance genes were  expressed.  How interesting was that?</p>
<p>Very interesting.  At  this point, I don’t have a clear explanation.   It’s something that we’ll look into in the future.  One of the nice things about  metatransciptomic studies is you can always go back and ask which organisms  express these genes?  What is their role in  this resistance?  But I haven’t had the  time to look in detail at those questions. </p>
<p style="color: rgb(27, 105, 1); font-weight: bold; font-size: 1.2em; line-height: 1.3em;">You spent a lot  of time and effort characterizing the role of a bacterium named TM7 in  disease.  Why TM7?</p>
<p>TM7 is associated with periodontal disease.  But these associations are based solely on  phylogenetic analysis and the high numbers of the bacterium in disease.  The reason is TM7 presently is a  non-cultivatable organism.</p>
<p style="color: rgb(27, 105, 1); font-weight: bold; font-size: 1.2em; line-height: 1.3em;">You can’t grow  it in the lab?</p>
<p>Exactly.  So, I  thought it would be interesting to look at the functions of TM7 in the real  microbial world using a metagenomic approach, and it worked.  We started making some sense of the potential  role of TM7 in disease or health.  </p>
<p style="color: rgb(27, 105, 1); font-weight: bold; font-size: 1.2em; line-height: 1.3em;">What did you  find?</p>
<p>We found that during severe periodontitis, TM7 expresses  certain genes that it doesn’t during health.   Some of these genes are involved in iron acquisition and others produce  putative virulence factors.  But, again,  we are still in the early stages of our research.  I wouldn’t make a big deal about these  findings yet.  </p>
<div style="width: 200px; float: right; margin-left: 10px;"><img width="194" height="125" alt="Electron microscopy image of an oral biofilm" src="/ScienceSpotlight/InterviewsbyTopic/PeriodontalDisease/PublishingImages/biofilm.jpg" style="padding-bottom: 10px;"/><p><span style="color: rgb(34, 34, 34); font-size: 11px;"><strong>The oral microbiofilm contains many species of bacteria.</strong></span></p></div>
<p style="color: rgb(27, 105, 1); font-weight: bold; font-size: 1.2em; line-height: 1.3em;">But the larger  point is you could at least begin to study TM7?</p>
<p>Yes.  These genomic  techniques allow you to peek into the functions of an organism without any  preconceived ideas about them.  I’m a  microbiologist by training, and, for me, it’s so interesting to gain a first  glimpse of what these uncultivable organisms might be doing in the real  world.  </p>

<p style="color: rgb(27, 105, 1); font-weight: bold; font-size: 1.2em; line-height: 1.3em;">Let’s jump to  your group’s second paper.  Based on some  earlier work and data from the first paper, you decided to look at a curious  effect that <em>P. gingivalis</em> seems to  have on the bacterium <em>Streptococcus mitis</em>.  Why?  </p>
<p><em>S. mitis</em> is an  early colonizer of a developing oral biofilm and abundant in healthy  microbiomes that don’t cause disease.  In  previous work in culture, we introduced <em>P.  gingivalis</em> and another periodontal pathogen called <em>Aggregatibacter actinomycetemicomitans</em> to see how our healthy  biofilm model reacted to the new arrivals.   We were surprised by what we saw.  </p>
<p style="color: rgb(27, 105, 1); font-weight: bold; font-size: 1.2em; line-height: 1.3em;">What was that?</p>
<p>First of all, we saw expression profiles of healthy organisms  changed completely.  We observed that  when those pathogens were added, <em>S. mitis</em> in particular started to upregulate the expression of enzymes called  transposases like crazy.  That caught my  attention.  At first, I thought the  transposases helped <em>S. mitis</em> mobilize, or shuffle, its DNA to rise to the challenge of living with its new  neighbors. But the high number of transposates was suspicious.  I mean, if you want to mobilize DNA, why  overdose on tranposases?  A colleague  suggested that it may have nothing to do with mobilization.  It might have everything to do with cell  death.  </p>
<p style="color: rgb(27, 105, 1); font-weight: bold; font-size: 1.2em; line-height: 1.3em;">Did you look  into that?</p>
<p>Yes.  What we found is  when we added <em>P. gingivalis</em> alone, it  caused <em>S. mitis</em> cells to die.  There is something that this bug does that  induces <em>S. mitis</em> cell death.  That’s interesting because, it is well known  that when you have a single-species biofilm that grows old, part of the  population kills itself in order to release DNA and other nutrients that allow  the others to survive.  What’s  interesting here is it’s not that <em>S.  mitis</em> senses that the biofilm is old.   The biofilm isn’t old at all.  But <em>P. gingivalis</em> does something – and we  still don’t know what that something is – that induces the death of <em>S. mitis </em>cells.  Ecologically, that has really big consequences.  If we can prove in the future that <em>P. gingivalis</em> makes other  organisms die—particularly health-associated bacteria, it may have important  implications for understanding how biofilms become diseased.  </p>
<div align="center">
<img width="596" height="261" alt="image of two bacteria viability assays, A and B" src="/ScienceSpotlight/InterviewsbyTopic/PeriodontalDisease/PublishingImages/live-dead-bacteria.png" style="padding-bottom: 10px;"/>
<p><span style="color: rgb(34, 34, 34); font-size: 11px;"><strong>A live-dead bacteria viability assay using confocal microscopy shows addition of <em>P gingivalis</em> to a healthy biofilm of <em>S. mitis </em>cells (A) causes them to die (B).</strong></span></p>
</div>
<p style="color: rgb(27, 105, 1); font-weight: bold; font-size: 1.2em; line-height: 1.3em;">It also adds  another piece to the “keystone pathogen” hypothesis.  Briefly, Hajishengalis and colleagues  proposed that <em>P. gingivalis</em> populates  biofilms in low abundance but hovers in the shadows to disrupt immune function  to its dysbiotic benefit.  So these data  suggest that <em>P. gingivalis</em> might  wreak havoc not only on the immune system but at least on some of the good bugs  in the biofilm?</p>
  <p>
That’s right.   Hajishengalis observed that the community changed.  We don’t know in which ways the community  changed or which set of signals the keystone pathogen uses to change the  community to its advantage.  The other  interesting thing is we still don’t know what the signal is to cause <em>S. mitis</em> cell death.  If we can somehow learn the language that  these organisms use to tell <em>S. mitis</em> to die, you can think of scenarios in which you can use those signals to kill  unwanted organisms instead of using antibiotics.  That’s an idea that I have, but first we must  identify this signal.  </p>
<p style="color: rgb(27, 105, 1); font-weight: bold; font-size: 1.2em; line-height: 1.3em;">But what’s  unique about the oral biofilm is the high density of cells and the species that  have specific cell-to-cell interactions.   How is that unique as opposed to biofilms in other parts of the body?  Is our understanding of the biofilm more  developed in the mouth?</p>
<p>Well, I think the density per square inch is even greater in  the gut.  But one of the advantages that  we have in the oral biofilm is that we have a wealth of historical knowledge  that other sites in the body don’t.  In  the mouth, we have a pretty good idea of its spatial, or at least its  sequential, colonization.  We know which  organisms may be important in periodontal disease.  In the gut, they’re still trying to piece  together which bugs are important and what the core community is in a normal  gut.  So, we are way ahead of most of the  other sites in the body.  </p>
<p style="color: rgb(27, 105, 1); font-weight: bold; font-size: 1.2em; line-height: 1.3em;">We haven’t  talked at all about the role of inflammation in driving chronic periodontitis.  Is immunity in your long-term research  picture? </p>
<p>It is.  My ultimate  goal is to understand immune function, too.   What happens to the host and its defenses when the disease is  active?  I want to try to make the link between  what we see in the bacterial community and the specific response of the host  under different conditions. I&#39;d also like to do a  dual analysis and glean a more comprehensive picture of what’s going on.  </p>
<p style="color: rgb(27, 105, 1); font-weight: bold; font-size: 1.2em; line-height: 1.3em;">In other words,  the bugs are the initiators, and inflammation is the promoter of chronic  disease?</p>
<p>Yes.  The final goal  may be to subclassify diseases according to the type of virulence that is  active at a site and its effects on immune function.  The idea being that periodontal diseases are  truly plural, not singular. But, above all, it would be to characterize these  functions.  If you see early expression  of these genes at a site in your mouth, you run the risk that it will advance  to periodontitis in weeks or months.  The  other thing that we did, based on our knowledge of these functions, is try to  devise new strategies to reduce disease.   That will happen when we know more about metagenomic function.  </p>
<p style="color: rgb(27, 105, 1); font-weight: bold; font-size: 1.2em; line-height: 1.3em;">And you’re getting  there.</p>
<p>Yes we are.  The tools  are available now to begin performing these types of analyses that generate  meaningful data.  There are just a  plethora of questions to ask in the laboratory.   I hope more people get involved in this research area to generate more  information to tell us what’s going on there.  </p>
<p style="color: rgb(27, 105, 1); font-weight: bold; font-size: 1.2em; line-height: 1.3em;">Thanks so much  for your time.<span style="color: rgb(51, 51, 51); font-size: 12px; line-height: 1.4em;">​​​​​​​​​​</span></p>